mgmtfn/k8splugin/driver.go

```diff
@@ -90,6 +92,26 @@ func netdGetNetwork(networkID string) (*mastercfg.CfgNetworkState, error) {
 	return nwCfg, nil
 }
 
+// netdGetEpg is a utility that reads the n/w oper state
+func netdGetEpg(epgName, tenantName string) (*mastercfg.EndpointGroupState, error) {
+	// Get hold of the state driver
```

Comment:
netd prefix is used for functions in netDriver.c. Ideally we should rename and move these to utils